Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,135 @@
+
+### CephFilesystemMirror CRD
+
+CephFilesystemMirror CRD is used by Rook to allow [creation](../CRDs/Shared-Filesystem/ceph-fs-mirror-crd.md) and updating the Ceph fs-mirror daemon.
+
+### FilesystemSubVolumeGroup CRD
+
+CephFilesystemMirror CRD is used by Rook to allow [creation](../CRDs/Shared-Filesystem/ceph-fs-subvolumegroup-crd.md) of Ceph Filesystem SubVolumeGroups.  
+
+### CephObjectStore CRD
+
+CephObjectStore CRD is used by Rook to allow [creation](https://rook.io/docs/rook/latest/CRDs/Object-Storage/ceph-object-store-crd/#example) and customization of object stores.
+
+### CephObjectStoreUser CRD
+
+CephObjectStoreUser CRD is used by Rook to allow creation and customization of object store users. For more information and examples refer to this [documentation](../CRDs/Object-Storage/ceph-object-store-user-crd.md).
+
+### CephObjectRealm CRD
+
+CephObjectRealm CRD is used by Rook to allow creation of a realm in a Ceph Object Multisite configuration. For more information and examples refer to this [documentation](../CRDs/Object-Storage/ceph-object-realm-crd.md).
+
+### CephObjectZoneGroup CRD
+
+CephObjectZoneGroup CRD is used by Rook to allow creation of zone groups in a Ceph Object Multisite configuration. For more information and examples refer to this [documentation](../CRDs/Object-Storage/ceph-object-zonegroup-crd.md).
+
+### CephObjectZone CRD
+
+CephObjectZone CRD is used by Rook to allow creation of zones in a ceph cluster for a Ceph Object Multisite configuration. For more information and examples refer to this [documentation](../CRDs/Object-Storage/ceph-object-zone-crd.md).
+
+
+### CephClient CRD
+
+CephClient CRD is used by Rook to allow [creation](https://rook.io/docs/rook/latest/CRDs/ceph-client-crd/) and updating clients.
+
+### CephNFS CRD
+
+CephNFS CRD is used by Rook to allow exporting NFS shares of a CephFilesystem or CephObjectStore through the CephNFS custom resource definition. For further information please refer to the example [here](https://rook.io/docs/rook/latest/CRDs/ceph-nfs-crd/#example).
+
+
+## Ceph
+
+Ceph is a distributed network storage and file system with distributed metadata management and POSIX semantics. 
+
+* [Ceph Monitor](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Monitor) (MON)
+* [Ceph Manager](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Manager) (MGR)
+* [Ceph Metadata Server](https://docs.ceph.com/en/latest/glossary/#term-MDS) (MDS)
+* [Object Storage Device](https://docs.ceph.com/en/latest/glossary/#term-OSD) (OSD)
+* [RADOS Block Device](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Block-Device) (RBD)
+* [Ceph Object Gateway](https://docs.ceph.com/en/latest/glossary/#term-Ceph-Object-Gateway) (RGW)
+
+See also the [Ceph Glossary](https://docs.ceph.com/en/latest/glossary/) for more definitions.
```

Comment:
How about combining this with the other sentence above?

Reply:
@travisn sure done.